src/browser/declarativewebutils.cpp

```diff
@@ -32,7 +32,6 @@
 
 #include <math.h>
 #include "declarativewebutils.h"
-#include "qmozcontext.h"
```

Comment:
gah, github removes angle brackets.  #include webenginesettings.h or #include webengine.h, is what i meant.